doc/SUMMARY.md

```diff
@@ -44,4 +45,4 @@
     - [api]() (to come)
     - [admin]() (to come)
     - [telemetry]() (to come)
-    - [consul]() (to come)
\ No newline at end of file
+    - [consul]() (to come)
```

Comment:
I'm not entirely sure I understand what this diff is but I'll try to remove it in a bit